### DISCUSSION TITLE
Add ability to enforce a set star range in multi rooms

### DISCUSSION THREAD

**Discussion by ggliv** [ppy/osu#18994] [Ideas] [answered]
The new difficulty range filter in song select got me thinking about how a similar setting could work in multiplayer. Multi rooms often have target star ranges in their names, but in my experience, these limits are frequently ignored in open queue modes. This forces hosts to either remove a person's map choice (which can be frustrating for both parties) or go along with a map that may be harder than lobby participants are comfortable with. Adding a setting to allow lobby hosts to enforce a set star range for maps would help alleviate this problem and encourage more players to join lobbies with the assurance that they'll be playing beatmaps within that set range.

I'm not sure how this range would be communicated in lobby select or in rooms, as lobby cards already display a range for beatmaps that have been added to the queue.
![image](https://user-images.githubusercontent.com/47010459/177050593-0cdb9a5c-b59c-46ea-a1db-fb2ccbb88787.png)
The locked-in range could displace this range, but that isn't fantastic UX. As far as I'm aware, no designs exist for something like this exist.

All in all, does anyone have any additional ideas for how this should be implemented? Would it be a welcome addition?

**Comment by Theighlin** [comment, marked as answer]
Duplicate of #15006. Though maybe i would answer them both and track in #16790? Though it goes against the one discussion per issue way of doing things...

**Reply by ggliv** [reply]
Surprised my search didn’t pull either of these up. Thanks for the links.
